noggin/utility/pagination.py

```diff
@@ -30,7 +30,7 @@ def page_url(self, page_number):
         qs = dict(request.args)
         qs.update({"page_size": self.page_size, "page_number": page_number})
         qs = "&".join(f"{k}={v}" for k, v in qs.items())
-        return f"{request.path}?{qs}"
+        return f"{request.script_root}{request.path}?{qs}"
```

Comment:
I was looking into this and I tried it with ``request.base_url`` anything against using the full path directly?

Reply:
It would work too, it's just that `base_url` would give the full URL including scheme and hostname, and we don't need that. But both would work really. I usually avoid full URLs for internal links because I've seen it fail too many times when there's a proxy that does SSL termination in front of the app, but in our case that possibility of failure is already taken care of, so both ways are equivalent I think.